napari/_qt/layer_controls/qt_image_controls.py

```diff
@@ -54,22 +39,53 @@ class QtImageControls(QtBaseImageControls):
         Button to pan/zoom shapes layer.
     transform_button : napari._qt.widgets.qt_mode_button.QtModeRadioButton
         Button to transform shapes layer.
-    attenuationSlider : qtpy.QtWidgets.QSlider
-        Slider controlling attenuation rate for `attenuated_mip` mode.
-    attenuationLabel : qtpy.QtWidgets.QLabel
-        Label for the attenuation slider widget.
+
+    Controls attributes
+    -------------------
```

Comment:
We currently do not render docs for `QtImageControls` etc, But I expect that if we would like to do this, then this section may not be rendered correctly. 

I'm not sure if we should worry about it, but if yes, then we should open a proper issue in docs report on merge.

Reply:
I added that custom section to have a little more clear division of the docstring while I was doing the changes but it should be fine to remove it now. Leaving only `Parameters` and `Attributes` sections should prevent worrying about possible things not being rendered correctly over the docs, right?